src/state/store/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import rootReducer from '../reducers/rootReducer'
+
```

Comment:
could get rid of this file and move content to index.js

Reply:
Agree, updated